### PR TITLE
docs: add Core Mobile, Core Extension and Avalanche JS v4.0 audits

### PR DESCRIPTION
**Pull request by gergelylovas**

### CODE REVIEW

**Review by akly — APPROVED** [review]

**Review by akly — CHANGES_REQUESTED** [review]
Please upload the updated Mobile Wallet audit report

File: core/Core_Mobile_Wallet_Audit_Least_Authority_November_2023.pdf
Comment (akly) [inline core/Core_Mobile_Wallet_Audit_Least_Authority_November_2023.pdf:1]: Please update this file to the latest version dated 2024 Apr 12

Comment (akly) [reply core/Core_Mobile_Wallet_Audit_Least_Authority_November_2023.pdf:1]: Thanks. Please also update file name to ..._April_2024.pdf .

File: README.md [hunk 1]
```diff
@@ -2,29 +2,40 @@
...
+
+## Core
+
+### Core Extension
+
+- [Least Authority Audit (December 15th 2023)](./core/Core_Extension_Audit_Least_Authority_December_2023.pdf)
+
+### Core Mobile
+
+- [Least Authority Audit (November 30th 2023)](./core/Core_Mobile_Wallet_Audit_Least_Authority_November_2023.pdf)
```
Comment (akly) [inline README.md]: Please update filename to match new date of the report (April 2024)

**Review by akly — APPROVED** [review]
Approved updated docs

**Comment by akly** [comment]
@JacobEverly can you please merge this PR if you're happy to do so?

**Review by JacobEverly — APPROVED** [review]

**Review by StephenButtolph — APPROVED** [review]

**Review by Tawabsadat — APPROVED** [review]

**Pull request merged**